src/main/java/com/powsybl/openloadflow/sa/AcSecurityAnalysis.java

```diff
@@ -84,10 +84,11 @@ SecurityAnalysisReport runSync(String workingVariantId, SecurityAnalysisParamete
         // load contingencies
         List<Contingency> contingencies = contingenciesProvider.getContingencies(network);
         // try to find all switches impacted by at least one contingency and for each contingency the branches impacted
+        Set<String> allBusIdsToLose = new HashSet<>();
         List<PropagatedContingency> propagatedContingencies = PropagatedContingency.createList(network, contingencies, allSwitchesToOpen,
-                securityAnalysisParametersExt.isContingencyPropagation());
+                allBusIdsToLose, securityAnalysisParametersExt.isContingencyPropagation());
 
-        boolean breakers = !(allSwitchesToOpen.isEmpty() && allSwitchesToClose.isEmpty());
+        boolean breakers = !(allSwitchesToOpen.isEmpty() && allSwitchesToClose.isEmpty() && allBusIdsToLose.isEmpty());
```

Comment:
So we need to load network from bus/breaker view because bus ids contingency are base on bus/breaker view ?

Reply:
Yes it is my first idea, but what do you think?

Reply:
Yes ok